spec/features/user/user_cant_view_other_user_page_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'user goes to other user page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.